src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import { StyleSheet, View, Text, Image } from "react-native";
+
+import { styled } from "nativewind";
+
+import { AvatarType } from "./Avatar.types";
+import { getAvatarColor, getAvatarInitials } from "../../utils";
+
+const Avatar: React.FC<AvatarType> = ({ username, image }) => {
+	const [backgroundColor, setBackgroundColor] = React.useState("");
+
+	let avatarStyle = { ...style.avatarBase, backgroundColor };
+
+	React.useEffect(() => {
+		setBackgroundColor(getAvatarColor());
```

Comment:
this doesn't need to be in a use effect. I think it can be a const that lives outside of the component, since it'll be set once, only when the component is created and should never change right?